Magento_Theme/templates/html/slider.phtml

```diff
@@ -60,19 +60,19 @@ $sliderContentAfter = $block->getContentAfter();
             slider--loading
             <?= $block->escapeHtmlAttr($sliderClass) ?>
         "
-        data-mobile-first="<?= $sliderMobileFirst ?>"
-        data-infinite="<?= $sliderInfinite ?>"
-        data-center-mode="<?= $sliderCenterMode ?>"
-        data-dots="<?= $sliderDots ?>"
-        data-autoplay="<?= $sliderAutoplay ?>"
-        data-autoplay-speed="<?= $sliderAutoplaySpeed ?>"
-        data-pause-on-focus="<?= $sliderPauseOnFocus ?>"
-        data-pause-on-hover="<?= $sliderPauseOnHover ?>"
-        data-slides-to-show="<?= $sliderSlidesToShow ?>"
-        data-slides-to-scroll="<?= $sliderSlidesToScroll ?>"
-        data-responsive-config="<?= $sliderResponsiveConfig ?>"
-        data-swipe-to-slide="<?= $sliderSwipeToSlide ?>"
-        data-element-slides="<?= $slidesElement ?>"
+        data-mobile-first="<?= $block->escapeHtmlAttr($sliderMobileFirst) ?>"
+        data-infinite="<?= $block->escapeHtmlAttr($sliderInfinite) ?>"
+        data-center-mode="<?= $block->escapeHtmlAttr($sliderCenterMode) ?>"
+        data-dots="<?= $block->escapeHtmlAttr($sliderDots) ?>"
+        data-autoplay="<?= $block->escapeHtmlAttr($sliderAutoplay) ?>"
+        data-autoplay-speed="<?= $block->escapeHtmlAttr($sliderAutoplaySpeed) ?>"
+        data-pause-on-focus="<?= $block->escapeHtmlAttr($sliderPauseOnFocus) ?>"
+        data-pause-on-hover="<?= $block->escapeHtmlAttr($sliderPauseOnHover) ?>"
+        data-slides-to-show="<?= $block->escapeHtmlAttr($sliderSlidesToShow) ?>"
+        data-slides-to-scroll="<?= $block->escapeHtmlAttr($sliderSlidesToScroll) ?>"
+        data-responsive-config="<?= $block->escapeHtmlAttr($sliderResponsiveConfig) ?>"
+        data-swipe-to-slide="<?= $block->escapeHtmlAttr($sliderSwipeToSlide) ?>"
+        data-element-slides="<?= $block->escapeHtmlAttr($slidesElement) ?>"
         data-dots-class="slider__dots"
```

Comment:
Please test it on frontend. Especially some slider with defined responsive config.

Reply:
Changed `data-responsive-config` to `escapeHtml`.